oap-server/server-bootstrap/src/main/resources/component-libraries.yml

```diff
@@ -493,4 +496,5 @@ Component-Server-Mappings:
   Mysqli: Mysql
   influxdb-java: InfluxDB
   Predis: Redis
-  PyMysql: Mysql
\ No newline at end of file
+  PyMysql: Mysql
+  Hase: HBase
```

Comment:
What is this? If they are same, you don't need mapping.

Reply:
fixed

Reply:
Please try https://github.com/apache/skywalking/pull/5232, you should not need this once the bug fixed.